### PR TITLE
Kicks away mission loot in the teeth

### PR DESCRIPTION
**Pull request by Detective-Google**
## About The Pull Request

Goes about and kills most of the super OP epic gamer loot in most away missions.
Also moves the gateway on every away mission to be next to it's spawn, meaning no more getting players trapped in a gauntlet to get back to the station and dying and killing the pop
## Why It's Good For The Game

hopefully these can be run again

## Changelog
:cl:
balance: away missions
/:cl:
wildwest is still shit and should probably stay disabled
spacebattle was a crime against god and it is now **dead**
also Snowdin has kinda fucked loot-lists which probably warrant another future looking into

### CODE REVIEW

**Comment by Detective-Google** [comment]
it was clear some of these maps hadn't had mapmerger run on them in a _long long time_

**Comment by MrJWhit** [comment]
Thank god, that was the main reason why I didn't load away missions during secret rounds.

File: code/modules/awaymissions/mission_code/snowdin.dm [hunk 1]
```diff
@@ -475,65 +475,39 @@
...
 				/obj/item/borg/upgrade/disablercooler = 7)
 
 
 /obj/effect/spawner/lootdrop/snowdin/dungeonheavy
 	name = "dungeon heavy"
-	loot = list(/obj/item/twohanded/singularityhammer = 25,
-				/obj/item/twohanded/mjollnir = 10,
-				/obj/item/twohanded/fireaxe = 25,
+	loot = list(/obj/item/twohanded/fireaxe = 25,
 				/obj/item/organ/brain/alien = 17,
```
Comment (Arturlang) [inline code/modules/awaymissions/mission_code/snowdin.dm:506]: no reason to keep this around

**Comment by Arturlang** [comment]
This pr basicallu removes all of the usefull loot without substituting it with more reasobable loot, only mostly useless stuff

**Comment by Detective-Google** [comment]
> This pr basicallu removes all of the usefull loot without substituting it with more reasobable loot, only mostly useless stuff

yes the whole point is to over-kill dunk the loot so they can be played without fucking over game balance until someone goes through and does a more thorough look over

**Pull request merged**